app/views/components/fields/field_accessible_autocomplete_with_header.scala.html

```diff
@@ -63,8 +63,12 @@
 
     @allErrors.map { error => <span id="error-message-@{field.name}-input" class="govuk-error-message">@error</span> }
 
-    @hintParagraphs.map { hint =>
-        <p class="govuk-hint" id="@(fieldNameWithoutBrackets)-hint">@hint</p>
+    @if(hintParagraphs) {
```

Comment:
I have to admit that I am surprised this is working, since Scala requires `Boolean` type inside `if` statement. I'm guessing this is due to having Twirl here rather than Scala.